plugins/modules/ovh_ip_failover.py

```diff
@@ -158,7 +158,7 @@ def waitForTaskDone(client, name, taskId, timeout):
         task = client.get('/ip/{0}/task/{1}'.format(quote_plus(name), taskId))
         if task['status'] == 'done':
             return True
-        time.sleep(5)  # Delay for 5 sec because it's long to wait completion, do not harass the API
+        time.sleep(5)  # Delay for 5 sec because it is long to wait completion, do not harass the API
```

Comment:
Sentence sounds weird to my ears. I get what they mean but maybe:
```suggestion
        time.sleep(5)  # Delay for 5 sec not to harass the API
```
?

Reply:
Shouldn't it be "to not" instead of "not to"?

Reply:
I believe they're both right. "Not to mention that... blablabla" is quite a common construct.

Reply:
According to https://twominenglish.com/to-not-or-not-to/:

> However, in some cases, especially when emphasizing the action you’re avoiding, “to not” fits better.

which I think applies here.

Reply:
```suggestion
        time.sleep(5)  # Delay for 5 sec to not harass the API
```